config/locales/gobierto_dashboards/views/ca.yml

```diff
@@ -156,6 +157,17 @@ ca:
           subsidies: Subvencions
           subsidies_for: subvencions per import de
         title: Subvencions
+      visualizations:
+        by_amount: per import de
+        contracts: Adjudicacions
+        difference_import: Diferència Import
```

Comment:
Capitalize?

Reply:
In this case, no.
![Screenshot 2020-11-25 at 16 09 41](https://user-images.githubusercontent.com/2649175/100245748-afdb7000-2f38-11eb-9e10-68bf3ec40242.png)

Reply:
I was pointing `difference_import: Diferència Import`